### PR TITLE
Tabs and Basic homepage

### PR DESCRIPTION
**Pull request by OlejukM**
Added tabs and description to each lesson.
Homepage was edited now it's close to design.

### CODE REVIEW

File: src/app/homepage/homepage.component.html [hunk 1]
```diff
@@ -3,13 +3,17 @@
...
-                    <h2 class="text-uppercase font-weight-bold font-italic">{{title}}</h2>
+                    <h2 class="text-uppercase font-weight-bold">{{ title }}</h2>
+                    <h3 class="text-uppercase font-weight-bold font-italic">{{ subTitle }}</h3>
                 </div>
-                <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 text-center">
-                    <app-modal-login></app-modal-login>
+                <!--<div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 text-center">-->
+                    <!--<app-modal-login></app-modal-login>-->
+                <!--</div>-->
+                <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 align-items">
```
Comment (Vagabonde) [inline src/app/homepage/homepage.component.html]: just col-12?

Comment (OlejukM) [reply src/app/homepage/homepage.component.html]: minified👍

File: src/app/group/timeline/timeline.component.ts [hunk 2]
```diff
@@ -9,9 +9,10 @@ import { TASKS } from './mock-tasks'
 })
 
 export class TimelineComponent implements OnInit {
-
   tasks = TASKS;
   selectedTask: Task;
+  selectedInfo: Task;
+  tabs = 'Hello';
```
Comment (reshetniktati) [inline src/app/group/timeline/timeline.component.ts]: what is this variable needed (tabs) for?

File: src/app/app-routing.module.ts [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
+import {GroupsComponent} from "./groups/groups.component";
 import {GroupComponent} from "./group/group.component";
 import {HomepageComponent} from "./homepage/homepage.component";
 import {PageNotFoundComponent} from './page-not-found/page-not-found.component';
 import {AuthGuard} from './core/auth.guard';
 
 const routes: Routes = [
-    {path: '', component: HomepageComponent},
-    {path: 'groups', component: GroupsComponent, canActivate: [AuthGuard]},
+    {path: '', component: GroupComponent},
```
[the remark this answers is not in the document]
Comment (OlejukM) [reply src/app/app-routing.module.ts]: done 👍

File: src/app/group/timeline/timeline.component.ts (hunk 2 above)
Comment (OlejukM) [reply src/app/group/timeline/timeline.component.ts]: They were unused, thanks.
Fixed 👍

**Pull request merged**